### PR TITLE
feat: Create Order model

### PR DESCRIPTION
**Pull request by Maui21**
Check my code

### CODE REVIEW

**Comment by Maui21** [comment]
closes #3

File: db/models/order.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  city: {
+    type: STRING,
+  },
+  state: {
+    type: ENUM(...states),
+  },
+  zip: {
+    type: STRING,
+    allowNull: true,
+    validate: {
```
Comment (benjmac) [inline db/models/order.js]: Won't be able to validate here. Should do on front end and en put route request

File: db/models/order.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      len: [5]
+    }
+  },
+  shippingMethod: ENUM('Express', 'Standard'),
+  isPurchase: {
```
Comment (benjmac) [inline db/models/order.js]: isPurchased? as in past tense? Idk...

File: db/models/order.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    type: BOOLEAN,
+    defaultValue: false
+  },
```
Comment (benjmac) [inline db/models/order.js]: Date we be when it's created, not actually when the purchase is made. Will probably have to update that as well once the front end request is finally made to purchase.

File: db/models/order.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    fullAddress: function() {
+      return `${this.address}\n${this.city} ${this.state}\n${this.zip})`
+    }
+  }
+})
+
+module.exports.associations = (Order, {User, Magnet}) => {
+  Order.belongsTo(User)
+  Order.hasMany(Magnet, {as: 'Magnets'})
+}
```
Comment (benjmac) [inline db/models/order.js]: Shouldn't need as magents, will create that name itself

**Pull request merged**